Anime-App/Feature/Anime/ViewModel/AnimeViewModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AnimeViewModel.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 09/08/2023.
+//
+
+import Foundation
+
+class AnimeViewModel {
+    func getAnime(completion: @escaping (Result<[Anime], Error>) -> Void) {
+        // get request
+        guard let url = URL(string: "https://kitsu.io/api/edge/anime") else { return }
```

Comment:
Can you organise the base url and endpoints in a separate file to clean things up